packages/communication-react/CHANGELOG.stable.md

```diff
@@ -1,9 +1,49 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
+
+Mon, 06 Nov 2023 16:44:11 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.10.0-beta.1...@azure/communication-react_v1.10.0)
```

Comment:
We missed this in the grooming. This should be:
`[Compare changes](https://github.com/azure/communication-ui-library/compare/1.10.0-beta.1...1.10.0)`

Reply:
shouldn't we really be comparing against 1.9.0? @mgamis-msft

Reply:
Yeah that makes sense to me. We may need to change something in the beachball config to do this from now on. In the previous releases we have been comparing to the previous beta.